### DISCUSSION TITLE
What replaced the Financial class?

### DISCUSSION THREAD

**Discussion by rafaelmiti** [PHPOffice/PhpSpreadsheet#2322] [Q&A] [answered]
The class is deprecated. What replaced it?

**Comment by oleibman** [comment, marked as answer]
Most of the Calculation classes were refactored to move from one massive module to many smaller and easier-to-maintain modules. In the case of Financial, there is now Financial/Amortization (function AMORDEGRC and AMORLINC), Financial/Coupons (COUPDAYBS, COUPDAYS, COUPDAYSNC, COUPNCD, COUPNUM, COUPPCD), etc. There is even some second- and third-level refactoring, e.g. Financial/Cashflow/Single (FVSCHEDULE, PVDURATION, and RRI). The stub that remains as Calculation/Financial.php should have entries for all the old functions with a reference to where they now reside (look for `@see` comments).
